app/ns-framework-modules-category/application-settings/overview.md

```diff
@@ -1,2 +1,2 @@
 The Application Settings module is used to store strings, booleans and numbers in built-in key/value store.
-Uses `SharedPreferences` on Android and `NSUserDefaults` on iOS.
\ No newline at end of file
+The module uses `SharedPreferences` on Android and `NSUserDefaults` on iOS. The application-settings is suitable for tasks like saving and retriving small portions of custom values from the device’s local storage.  Storing to local storage means that you can reuse the key-value pairs after the applicaiton is restarted (e.g. remember if a user has logged in orsave user UI preferences).
```

Comment:
There is an additional space before "Storing to local..." that you can remove.
orsave -> or save
applicaiton -> application :)